internals/processes.py

```diff
@@ -274,7 +274,7 @@ def process_to_dict(process):
       [],
       [],
       core_enums.INTENT_SHIP, core_enums.INTENT_SHIPPED,
-      stage_type=core_enums.STAGE_BLINK_SHIPPED),
+      stage_type=core_enums.STAGE_BLINK_SHIPPING),
```

Comment:
Should this whole `ProcessStage` be deleted?  (Same for the other 2)

Reply:
The problem here is that if we outright remove this, features that have set this stage as active will not see "Revisit" options to edit their feature, because it the previous stages will not register as "prior" stages. We could work to make this change, but I also added a fix in [this commit](https://github.com/GoogleChrome/chromium-dashboard/pull/2569/commits/c9a6e55304f1cc42a312dee6a8e32621c99bbaeb) which only renders process stages that match actual stage entities. Maybe this is a better quick fix for now?